saleor/tax/utils.py

```diff
@@ -179,6 +179,11 @@ def normalize_tax_rate_for_db(tax_rate: Decimal) -> Decimal:
     return tax_rate / 100
 
 
+def denormalize_tax_rate_from_db(tax_rate: Decimal) -> Decimal:
+    # Revert results of `normalize_tax_rate_for_db`.
+    return tax_rate * 100
```

Comment:
Question: shouldn't we apply a `decimal.quantize` here?
```
In [2]: Decimal(0.22)*100
Out[2]: Decimal('22.00000000000000011102230246')
```

Reply:
Hmm I think this is a good idea.

Reply:
I'm checking this and it's not `Decimal(0.22)` but `Decimal("0.22")`. Any multiplication on that seems to give correct values:
```
ipdb> order.shipping_tax_rate
Decimal('0.2300')
ipdb> order.shipping_tax_rate * 100
Decimal('23.0000')
```

Reply:
We don't convert floats to decimals, there is no need to do any quantization or rounding here.